spec/features/users/regenerate_personal_key_spec.rb

```diff
@@ -22,7 +22,7 @@
         old_code = user.personal_key
 
         click_button t('account.links.regenerate_personal_key')
-
+        
```

Comment:
Nit here that rubocop should have seen, but looks like there's some trailing whitespace here.

Reply:
I know I can get sublime to detect trailing whitespace...really need to turn that on. 

Fixed. Thanks!